Example/FinancialConnections Example/FinancialConnectionsUITests/FinancialConnectionsNetworkingUITests.swift

```diff
@@ -26,34 +26,16 @@ final class FinancialConnectionsNetworkingUITests: XCTestCase {
     }
 
     private func executeNativeNetworkingTestModeSignUpFlowTest(emailAddress: String) {
-        let app = XCUIApplication.fc_launch()
+        let app = XCUIApplication.fc_launch(
+            playgroundConfigurationString:
+"""
+{"use_case":"payment_intent","sdk_type":"native","test_mode":true,"merchant":"networking","payment_method_permission":true,"email":""}
+"""
```

Comment:
this would be better if it was type-safe (ex. create configuration using a struct), but there's other more pressing issues, so it will do until there's some extra time to improve again